194: Fixing previous and next page link

abhishek44sharma opened the pull request:
194: Fixing previous and next page link

v0lkan commented:
Some of that navigation was intentional; 

I’ll check them and comment on each of them.

v0lkan commented on docs/README.md:
```diff
@@ -13,7 +13,7 @@ title: VMWare Secrets Manager for Cloud-Native Apps
 layout: post
 permalink: /
 next_url: /docs/community/
-prev_url: /docs/showcase/
```
Please keep this as `/docs/showcase` ; it gives like a circular navigation, and jumping to "showcase" by mistake and seeing lots of awesome videos, would be a lucky mistake :) .

v0lkan commented on docs/_pages/0000-community.md:
```diff
@@ -11,9 +11,9 @@
-next_url: /docs/navigation/
 prev_url: /
 permalink: /docs/community/
+next_url: /docs/contact/
```
+1

v0lkan commented on docs/_pages/0001-contact.md:
```diff
@@ -11,9 +11,9 @@
-prev_url: /
+prev_url: /docs/community/
 permalink: /docs/contact/
+next_url: /docs/maintainers/
```
+1

v0lkan commented on docs/_pages/0002-maintainers.md:
```diff
@@ -11,9 +11,9 @@
-prev_url: /
+prev_url: /docs/contact/
 permalink: /docs/maintainers/
+next_url: /docs/changelog/
```
I have mixed feelings; both going to `/docs/navigation/` and going to `/docs/changelog` have merit:

One of them means that this is a non-critical section and we are jumping to the meat of the content.

And the other would mean we are following things in sequential order.

I think it’s okay to keep it as `/docs/changelog` though. The user can skip a few pages further by pressing "next" if they are not interested in the content anyway.

abhishek44sharma replied:
I agree to the point of skipping Changelog and Roadmap for someone who's new taking look at the project.
But then this might look like a bug if it doesn't go to next page in sequence.
Maybe we can think of placing Changelog and Roadmap to another location, where it's easier for needy to go.

abhishek44sharma replied:
Leaving it to you @v0lkan to decide on above, we can create an issue if we want to do above.

abhishek44sharma replied:
Suggestion 💡: What do you think keeping Changelog and Roadmap along with Release Management?
So it doesn't get in the way for a new onlooker.

v0lkan commented on docs/_pages/0003-changelog.md:
```diff
@@ -11,9 +11,9 @@
-prev_url: /
+prev_url: /docs/maintainers/
 permalink: /docs/changelog/
+next_url: /docs/roadmap/
```
+1

v0lkan commented on docs/_pages/0004-roadmap.md:
```diff
@@ -11,9 +11,9 @@
-prev_url: /
+prev_url: /docs/changelog/
 permalink: /docs/roadmap/
+next_url: /docs/navigation/
```
+1

v0lkan commented on docs/_pages/0020-preface.md:
```diff
@@ -11,9 +11,9 @@
-prev_url: /docs/community/
+prev_url: /docs/roadmap/
 permalink: /docs/navigation/
+next_url: /docs/about/
```
Also +1 for the consistent `prevurl => permalink => nexturl` structure. 

it will help diff/merging and copy/pasting less error prone when/if we create more such pages.

v0lkan commented on docs/_pages/0250-showcase.md:
```diff
@@ -11,9 +11,9 @@
-next_url: /
 prev_url: /docs/use-case-transformation/
 permalink: /docs/showcase/
+next_url: /docs/release-management/
```
I would leave it at `/`.

Release management is more like reference material for 2-3 people who actually cut releases.

If the reader wants, they can just click the link on the sidebar.

abhishek44sharma replied:
Agree. 👍

v0lkan commented on docs/_pages/0980-release-management.md:
```diff
@@ -11,9 +11,9 @@
 layout: post
-next_url: /
-prev_url: /
+prev_url: /docs/showcase/
```
change `prev_url: /` please.

v0lkan reviewed (changes requested):
I have made some changes; to make navigation more “circular” and skip relatively less-useful parts such as privacy policy and release management instructions.

abhishek44sharma commented:
> I have made some changes; to make navigation more “circular” and skip relatively less-useful parts such as privacy policy and release management instructions.

Addressed review comment @v0lkan

v0lkan commented:
Approved.
